src/pslLint/mutliLineDeclare.ts

```diff
@@ -0,0 +1,69 @@
+import { MethodRule, PslDocument, Method, Diagnostic, DiagnosticSeverity, getTokens} from "./api";
+import { Member, NON_TYPE_MODIFIERS } from "../parser/parser";
+
+export class MultiLineDeclare implements MethodRule {
+
+    ruleName = MultiLineDeclare.name;
+
+	report(pslDocument: PslDocument, method: Method): Diagnostic[] {
+        
+        let diagnostics: Diagnostic[] = [];
+        let prevDeclaration: Member;
+
+        for (let i = 0; i < method.declarations.length; i++) {
+            let reportAllVariables: boolean = false;
+            let curDeclaration = method.declarations[i];
+            let fullLine = pslDocument.getTextAtLine(curDeclaration.id.position.line);
+            
+            if ((((fullLine.indexOf("=") > fullLine.indexOf(",")) && (fullLine.indexOf(",") > -1)) || ((fullLine.indexOf(",") > fullLine.indexOf("=")) && (fullLine.indexOf("=") > -1))) && !(prevDeclaration && prevDeclaration.id.position.line === curDeclaration.id.position.line)) {
```

Comment:
if I add only this condition, fullLine.includes('=') && fullLine.includes(','). It will be go and check most of the code. for example: for all type RecordDEP dep=getrecord(****) code eventhough it decared only one variable.